OpenEXR/exrmultipart/exrmultipart.cpp

```diff
@@ -85,7 +85,7 @@ using namespace OPENEXR_IMF_NAMESPACE;
 
 #if defined(ANDROID) || defined(__ANDROID_API__)
     #define IMF_PATH_SEPARATOR "/"
-#elif defined(_WIN32) || defined(_WIN64) || defined(__MWERKS__) || defined(PLATFORM_WINDOWS)
+#elif defined(_WIN32) || defined(_WIN64) || defined(__MWERKS__)
```

Comment:
I bet we shouldn't have __MWERKS__ here any more. Not sure why this would have ever been the case.

Reply:
who knows, but you're probably right, removing them, then merging.